pkg/apis/componentconfig/v1alpha1/zz_generated.conversion.go

```diff
@@ -814,6 +814,7 @@ func autoConvert_v1alpha1_LeaderElectionConfiguration_To_componentconfig_LeaderE
 	out.LeaseDuration = in.LeaseDuration
 	out.RenewDeadline = in.RenewDeadline
 	out.RetryPeriod = in.RetryPeriod
+	out.LockType = in.LockType
```

Comment:
I was making this switch in openshift the other day and used:

https://github.com/openshift/origin/pull/14094/files#diff-3788994f990386b63387ece5c3b5dc33R1418 for the componentconfig equivalent and https://github.com/openshift/origin/pull/14094/files#diff-9eadaa7b7aac3bc9ea7ed27cb8908d52R86 for the CLI flag equivalent.